opentelemetry/ext/honeycomb/__init__.py

```diff
@@ -104,6 +129,7 @@ def _translate_to_hny(spans):
             d['trace.parent_id'] = trace_api.format_span_id(span.parent.span_id)[2:]
         # TODO: use sampling_decision attributes for sample rate.
         d.update(span.attributes)
+        d.update(span.resource.attributes)
```

Comment:
Let's flip lines 132 and 131 and have resource attributes come before span attributes, so that if there are conflicts the span attributes win out.

Reply:
Oh, good catch. Done.